test/runtests.jl

```diff
@@ -89,12 +89,11 @@ else
     @test stop(t) == nothing
     @test digital_input(t, dev*"/Port0/Line1") == nothing
     @test NIDAQ.start(t) == nothing
-    @test length(read(t, UInt8, 6)) == 12
```

Comment:
in 4e8ca31e8f you have a nice switch to exclude tests for boards that don't have clocks.  could we do something like that here?  somehow test `read` on boards no matter how many digital lines they have??

Reply:
If I recall correctly the problem here is `UInt8`.  I think I can change all `UInt8` reads to `UInt32` reads (and update the value we're comparing to in the test) and that should be safe for any device.  Is that what you mean?  Or do you mean to skip the test if the device has more than 8 lines in a port?

Reply:
changing to `UInt32` would be great if it works.  if not, add a switch to use `UInt8` or `UInt32` as appropriate.

Reply:
UInt32 seems to work fine, though I've only been able to test it on one device.  I can't access other devices until 2 weeks from now.  If you test and it works for you feel free to merge before then (pushing the new code now).